### PR TITLE
setroles

### PR DESCRIPTION
**Pull request by makosblade**

### CODE REVIEW

**Comment by makosblade** [comment]
@synecdoche-nh @Bloodyhacker

**Comment by makosblade** [comment]
@UnsungHero97 you too!

File: src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java [hunk 1]
```diff
@@ -29,10 +30,13 @@
  */
 public interface UserDirectoryApi {
     public static final String CONTROLLER          = "/directory";
+
     public static final String ID                  = "id";
     public static final String EMAIL               = "email";
     public static final String REALM               = "realm";
     public static final String USERNAME            = "username";
+    public static final String ROLE                   = "role";
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java]: fix spacing

File: src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java [hunk 2]
```diff
@@ -45,6 +49,10 @@
                                                                                           // disambiguate from get user
     public static final String EMAIL_PATH_WITH_DOT = "/" + EMAIL + "/{" + EMAIL + ":.+}";
 
+    public static final String USER_ID_PATH           = USER + ID_PATH;
+    public static final String ROLE_ID_PATH           = "{" + ROLE + "}";
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java]: I think this needs a `/` before the opening curly brace

Comment (makosblade) [inline src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java]: Nice catch!

File: src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java (hunk 1 above)
Comment (makosblade) [inline src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java]: willfix

**Comment by Bloodyhacker** [comment]
LGTM

**Comment by makosblade** [comment]
Thanks!

**Pull request merged**